src/main/kotlin/study/Languages.kt

```diff
@@ -0,0 +1,13 @@
+package study
+
+class Languages {
+
+    private var _values: MutableMap<String, Int> = mutableMapOf()
```

Comment:
Backing properties 👍 

https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties